T/TulipIndicators/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+using BinaryBuilder
+
+name = "TulipIndicators"
+version = v"0.9.1"
+hash = "85147c3db868dca9880f78b26de9f4fcecf95fb9" # last rev as of 2022-07-14
+
+sources = [
+	GitSource("https://github.com/TulipCharts/tulipindicators.git", hash)
+]
+
+deps = [
+	HostBuildDependency("Tcl_jll")
+]
+
+script = raw"""
+if ! command -v tclsh &> /dev/null
+then
+	cd ${host_bindir}
+	ln -s tclsh* tclsh
+fi
+cd $WORKSPACE/srcdir/tulipindicators
+make tiamalgamation.c
+gcc -fPIC -shared -Wall -Wextra -Wshadow -Wconversion -std=c99 -pedantic -O2 -g tiamalgamation.c -o libindicators.so
+mkdir -p ${prefix}/include ${libdir}
+mv -t ${libdir} libindicators.so
+"""
+
+platforms = supported_platforms()
+filter!(p -> !(Sys.iswindows(p) || Sys.isapple(p)), platforms) # TODO apple and windows
```

Comment:
Anything particular needed for macOS and Windows?

Reply:
Windows works with your suggested `cc` compile command. Mac should work too but I haven't tested

Reply:
Having a problem when I'm trying to test apple build locally:
```
  Downloaded artifact: GCCBootstrap-aarch64-apple-darwin20.v12.0.1-iains.x86_64-linux-musl.unpacked
ERROR: LoadError: Unable to automatically install 'GCCBootstrap-aarch64-apple-darwin20.v12.0.1-iains.x86_64-linux-musl.unpacked'
```
IDK if the remote repo would have this problem though, might just be me

Reply:
I removed the filter and it works on remote